sysvinit/freebsd/dsidentify.tmpl

```diff
@@ -0,0 +1,34 @@
+## template:jinja
+#!/bin/sh
+
+# PROVIDE: dsidentify
+# REQUIRE: mountcritlocal
```

Comment:
once we are correctly using `paths.run_dir` / `paths.get_runpath()` in the python code-base, we can start thinking about how to squeeze that into ds-identify itself, and then!, then we can depend on `var_run` instead of `mountcritlocal` here.

Reply:
Do you think this would be useful as a comment in the code? If so, we could use a [template comment](https://jinja.palletsprojects.com/en/3.0.x/templates/#comments) to avoid render it in the output file.